signals/main.py

```diff
@@ -13,6 +13,10 @@
 # Issues unit testing `main` due to click decorators.
 def run_main(schema, generator_name, data_models, core_data, project_name, api_url, save):
     schema = Schema(schema)
+
+    if not api_url.endswith('/'):
```

Comment:
I think this would be better done outside the `run_main` method for organization reasons. The click library gives you the ability to validate parameters that are sent in: http://click.pocoo.org/5/options/#callbacks-for-validation - That sounds like a good place to do it.

Also it'd be good to write a test for it.